test/i18n.api.global.js

```diff
@@ -1,4 +1,4 @@
-var i18n = require('../i18n'),
+var i18n = require('../index'),
```

Comment:
Alternately, I could change the `require('../i18n')` bits to `require('../i18n')()` instead of  `require('../index') in most spots, but I think that could take more memory to test since each spots that does this would be creating it's own i18n object.